src/planner/parabolic/HauserParabolicSmootherHelpers.cpp

```diff
@@ -13,172 +13,184 @@ namespace planner {
 {
 public:
-    SmootherFeasibilityCheckerBase(aikido::constraint::TestablePtr testable,
-                                   double checkResolution)
-        : testable_(std::move(testable)),          
-          checkResolution_(checkResolution),
-          statespace_(testable_->getStateSpace()),
-          interpolator_(statespace_)
+  SmootherFeasibilityCheckerBase(
+      aikido::constraint::TestablePtr testable, double checkResolution)
+    : testable_(std::move(testable))
+    , checkResolution_(checkResolution)
+    , statespace_(testable_->getStateSpace())
+    , interpolator_(statespace_)
+  {
+  }
+
+  virtual bool ConfigFeasible(ParabolicRamp::Vector const& x)
+  {
+    Eigen::VectorXd eigX = toEigen(x);
+    auto state = statespace_->createState();
+    statespace_->expMap(eigX, state);
+    return testable_->isSatisfied(state);
+  }
+
+  virtual bool SegmentFeasible(
+      ParabolicRamp::Vector const& a, ParabolicRamp::Vector const& b)
+  {
+    Eigen::VectorXd eigA = toEigen(a);
+    Eigen::VectorXd eigB = toEigen(b);
+
+    auto testState = statespace_->createState();
+    auto startState = statespace_->createState();
+    auto goalState = statespace_->createState();
+    statespace_->expMap(eigA, startState);
+    statespace_->expMap(eigB, goalState);
+
+    aikido::util::VanDerCorput vdc{1, false, false, checkResolution_};
+
+    for (const auto alpha : vdc)
     {
+      interpolator_.interpolate(startState, goalState, alpha, testState);
+      if (!testable_->isSatisfied(testState))
+      {
+        return false;
+      }
     }
-
-    virtual bool ConfigFeasible(ParabolicRamp::Vector const &x)
-    {
-      Eigen::VectorXd eigX = toEigen(x);
-      auto state = statespace_->createState();
-      statespace_->expMap(eigX, state);
-      return testable_->isSatisfied(state);
-    }
-
-    virtual bool SegmentFeasible(ParabolicRamp::Vector const &a,
-                                 ParabolicRamp::Vector const &b)
-    {
-       Eigen::VectorXd eigA = toEigen(a);
-       Eigen::VectorXd eigB = toEigen(b);
-
-       auto testState = statespace_->createState();
-       auto startState = statespace_->createState();
-       auto goalState = statespace_->createState();
-       statespace_->expMap(eigA, startState);
-       statespace_->expMap(eigB, goalState);
-
-       aikido::util::VanDerCorput vdc{1, false, false, checkResolution_};
-
-       for (const auto alpha : vdc)
-       {
-         interpolator_.interpolate(startState, goalState, alpha, testState);
-         if (!testable_->isSatisfied(testState))
-         {
-           return false;
-         }
-       }
-       return true;
-    }
+    return true;
+  }
 
 private:
-    aikido::constraint::TestablePtr          testable_;
-    double                                   checkResolution_;
-    aikido::statespace::StateSpacePtr        statespace_;
-    aikido::statespace::GeodesicInterpolator interpolator_;
+  aikido::constraint::TestablePtr testable_;
```

Comment:
Nit: Could you rename these private member variables using the pattern of `mX`? I believe we don't use the trailing underscore for member variables (even for private member variables).